manual/coding_standards.tex

```diff
@@ -0,0 +1,480 @@
+\section{QMCPACK Coding Standards}
+
+The following document presents best practices for the code at the current time. New development should follow these guidelines and contributors are expected to adhere to them for the sake of consistency in the code base. A well prepared PR should follow these standards before inclusion in the mainline. A clang format file can be found in ? that should be run to fix and check the code style whenever possible.
+
+Keep reformatting, feature, and bug commits separate whenever possible.  
+
+
+\section{Files}
+Each file should start with the header
+\lstset{language=C++,style=C++}
+\begin{lstlisting}
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2018 Jeongnim Kim and QMCPACK developers.
```

Comment:
Copyright year should be the current year when a file is created.  The year in the header should also be updated when the file is significantly modified.